src/main/resources/color.avsc

```diff
@@ -0,0 +1,7 @@
+{"namespace": "example.avro",
+ "type": "record",
+ "name": "Color",
```

Comment:
Detta var en middlertidig fil som jag senare tog bort. Trenger inte att ta den in igen :)